AppDashboard/templates/layouts/panel.html

```diff
@@ -0,0 +1,29 @@
+{% if page_info.is_admin_panel == is_user_cloud_admin or page_info.is_admin_panel is not defined %}
+<!-- FILE:templates/apps/dash.html -->
+<!-- this is a child template to display a panel in dash.html-->
+<!-- it takes parameters id and page_info from the parent -->
+<div class="panel panel-default" id="panel-{{ id }}">
+  <div class="title panel-heading gradient">
+    <h2 class="panel-title">
+      <span aria-hidden="true" class="icon16"></span>
+      <a data-toggle="collapse" href="#{{ id }}" class="collapsed">
+        {{ page_info.title }}
+      </a>
+    </h2>
+
+  </div><!--end .panel-heading-->
+  <div id="{{ id }}" class="panel-collapse collapse in">
+    <div class="panel-body">
+      <div class="content">
+        {% include page_info.template %}
+        <script type="text/javascript">
+          $(document).ready(function(){
+              $(".well").removeClass("well");
```

Comment:
The class isn't necessarily needed, its just a styling thing. I could definitely remove it in the templates. The class is currently used when the url is used rather than the panel.